astropy/io/votable/tree.py

```diff
@@ -66,15 +66,11 @@ def _resize(masked, new_size):
     """
     new_array = ma.zeros((new_size,), dtype=masked.dtype)
     length = min(len(masked), new_size)
-    try:
-        # Pre Numpy 1.10 way
-        new_array.data[:length] = masked.data[:length]
-    except TypeError:
-        # Numpy 1.10 and later
-        new_array[:length] = masked[:length]
-    else:
-        if length != 0:
-            new_array.mask[:length] = masked.mask[:length]
+
+    new_array[:length] = masked[:length]
+
+    if length != 0:
```

Comment:
This part can be removed too, since it was part of the `else` clause that only gets executed if the `try` succeeds.